planner/src/main/java/com/dask/sql/application/RelationalAlgebraGeneratorBuilder.java

```diff
@@ -14,11 +15,13 @@
 	private final String rootSchemaName;
 	private final List<DaskSchema> schemas;
 	private final boolean case_sensitive; // False if case should be ignored when comparing SQLNode(s)
+	private final ArrayList<RelOptRule> disabled_rules; // CBO rules that should be disabled for this context
 
-	public RelationalAlgebraGeneratorBuilder(final String rootSchemaName, final boolean case_sensitive) {
+	public RelationalAlgebraGeneratorBuilder(final String rootSchemaName, final boolean case_sensitive, final ArrayList<RelOptRule> disabled_rules) {
```

Comment:
This method is expecting an `ArrayList` of `RelOptRule` objects but what it would get from python is actually an arrayList of strings. Do you think it makes sense to add a utility that accepts the strings and creates RelOptRule objects from them?

Reply:
Completely. I was going to expose the `RelOptRule` inside of `java.py` but I agree it makes more since to just let Java handle that conversion and hide that mess from Python. I will add another commit doing that.